microshift.py

```diff
@@ -66,8 +67,14 @@ def generate_kickstart(rhel_number: str, uname_m: str) -> None:
     with open('pull_secret.json', 'r') as f_in:
         file_contents = f_in.read()
 
-    with open('/root/.ssh/id_rsa.pub', 'r') as ssh_in:
-        ssh_contents = ssh_in.read()
+    pub_files = glob.glob('/root/.ssh/*.pub')
+    if pub_files:
+        with open(pub_files[0], 'r') as ssh_in:
+            ssh_contents = ssh_in.read()
+    else:
+        ssh_contents = None
```

Comment:
Why set ssh_contents to None if we sys.exit(-1)

Reply:
Will remove